system/include/compat/emmintrin.h

```diff
@@ -1020,7 +1020,7 @@ _mm_cvttsd_si64(__m128d __a)
   // TODO: optimize
   float e = __a[0];
   if (isnan(e) || isinf(e) || e > LLONG_MAX || e < LLONG_MIN) return 0x8000000000000000LL;
-  long long x = llrint(e);
+  long long x = llrintf(e);
```

Comment:
Same issue here. This should be `llrint` and `fabs`.

Reply:
Oops that means `e` has the wrong type too (from #22893).  Fixed